modules/ducktests/tests/checks/utils/check_ignite_spec.py

```diff
@@ -81,17 +81,19 @@ def check_default_jvm_options__are_not_used__if_merge_with_default_is_false(serv
 
 def check_boolean_options__go_after_default_ones_and_overwrite_them__if_passed_via_jvm_opt(service):
     service.context.globals[JFR_ENABLED] = True
-    spec = IgniteApplicationSpec(service, jvm_opts="-XX:-UnlockCommercialFeatures")
-    assert "-XX:-UnlockCommercialFeatures" in spec.jvm_opts
-    assert "-XX:-UnlockCommercialFeatures" in spec.jvm_opts
-    assert spec.jvm_opts.index("-XX:-UnlockCommercialFeatures") >\
-           spec.jvm_opts.index("-XX:+UnlockCommercialFeatures")
+    spec = IgniteApplicationSpec(service, jvm_opts="-XX:-FlightRecorder")
+    assert "-XX:-FlightRecorder" in spec.jvm_opts
```

Comment:
We have here 2 identical asserts.

I think one of them should be replaced with the 
`assert "-XX:+FlightRecorder" in spec.jvm_opts`

Otherwise it doesn't make sense.